api/proto/evecommon/devmodelcommon.proto

```diff
@@ -22,6 +22,8 @@ enum PhyIoType {
   // enum 8 is reserved for backward compatibility with controller API
   PhyIoNVMEStorage = 9;
   PhyIoSATAStorage = 10;
+  PHY_IO_TYPE_PHY_IO_NET_ETH_PF = 11;
+  PHY_IO_TYPE_PHY_IO_NET_ETH_VF = 12;
```

Comment:
Maybe I didn't comment on this before on this particular PR, but when we add enum values to an existing enum definition we follow the existing style, not the one yetus is asking for, because it becomes hard to relate the different values in the code when they use different styles e.g., in if/else or case statements.
(Only alternative is to change the style of all of the existing enum values which, if we were to do it, should be done as a separate rename PR.)

So here please name them PhyIoNetEthPf and PhyIoNetEthVf.
I see that change will have an effect on comments as well, so please grep -r to make sure all have been changed.